test/protocol-tests/src/test/java/software/amazon/awssdk/protocol/tests/ResourceManagementTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.protocol.tests;
```

Comment:
Any reason to have a separate package for them?

Reply:
The tests in that package don't test anything related to the protocols so it made sense to me to use a separate test package.

Reply:
Interesting. We have a lot of tests we could migrate, then, theoretically, if those are supposed to just be serialization tests.

Reply:
Oooh you're right. +1 to migrating them over

Reply:
Instead of requiring them to test against a separate client with a separate module can we just have one test module that depends on all the protocol test clients and put all hand written verification tests there? It seems wasteful to duplicate a dummy client/model when the protocol test clients cover all use cases. +1 to separating the tests into a separate module though.

Reply:
Nice that sounds good to me. 👍